src/main/resources/templates/navbar.html

```diff
@@ -8,46 +8,46 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="#">TDD-Pingis</a>
+        <a class="navbar-brand" href="#">TDD-pingpong</a>
     </div>
-
+    
```

Comment:
Try to avoid introducing unnecessary whitespace changes. `git diff --check` is useful for checking these.